lib/elixir/lib/enum.ex

```diff
@@ -1408,7 +1408,14 @@ defmodule Enum do
 
   """
   @spec sum(t) :: number
-  def sum(enumerable) do
```

Comment:
Do you mean making the `def sum(first..last) do` and `def sum(last..first) do` match the same structure as the existing one by changing them to `def sum(enumerable) when enumerable = first..last do`? Or are you talking about something else entirely?